ios/browser/api/brave_shields/adblock_filter_list_catalog_entry.h

```diff
@@ -23,9 +23,6 @@ OBJC_EXPORT
 @property(readonly) bool firstPartyProtections;
 @property(readonly) uint8_t permissionMask;
 @property(readonly) NSString* componentId;
-@property(readonly) NSString* base64PublicKey;
```

Comment:
we might still want this one?

Reply:
It's just the iOS interface. I don't see why we need this exposed to iOS. But maybe I'll need it in the future. I don't know. I'll put it back